libraries/adaptive-expressions/src/expressionFunctions.ts

```diff
@@ -1005,16 +1008,31 @@ export class ExpressionFunctions {
         return {value: result, error};
     }
 
-    private static extractElement(expression: Expression, state: MemoryInterface): { value: any; error: string } {
+    private static wrapGetValue(state: MemoryInterface, path: string, options: Options): any {
+        let result = state.getValue(path);
+        if (result !== undefined) {
+            return result;
+        }
+
+        if(options && options.nullSubstitution !== undefined) {
```

Comment:
can you try to comment out options, and just leave options.nullSubstitution, see, where options might not exisits?

Reply:
> can you try to comment out options, and just leave options.nullSubstitution, see, where options might not exisits?

The reason is the initialization is not consistent with dotnet. Will fix.